app/modules/events/card.tsx

```diff
@@ -0,0 +1,15 @@
+import type { Event } from '@prisma/client';
+
+export function EventCard({ name, date, description, imgUrl, imgAlt, location }: Event) {
+  return (
+    <div className="grid grid-cols-3 gap-8 p-4 bg-primary text-white rounded-xl">
+      <div>{imgUrl && imgAlt && <img src={imgUrl} alt={imgAlt}></img>}</div>
+      <div className="col-span-2">
+        <h1 className="font-bold text-3xl">{name}</h1>
```

Comment:
Suggest to change the `h1` tag to an `h2`. Typically there should only be one `h1` per page.